### PR TITLE
Updated definitions

### LINKED ISSUE #40: TSM 4.10 string changes
**Is your feature request related to a problem? Please describe.**
There’s a new version of tsm coming out that has new price sources to support. 

From: https://blog.tradeskillmaster.com/tsm-4-10-change-log/
Price Sources
There are new operation-based price sources available:
auctioningopmin, auctioningopmax, auctioningopnormal, shoppingmax, sniperopbelow
A new SmartAvgBuy price source has been added alongside the existing AvgBuy price source to replace the previous “Use Smart Average for Purchase Price” setting.
A new SaleRate price source has been added which represents your personal sale rate.
A new NumInventory price source has been added to represent your total inventory quantity.

### PR DESCRIPTION
**Pull request by jaredycalves**
* Added new sources from TSM 4.10
* Changed some texts

Closes #40

### CODE REVIEW

File: src/lib/definitions.js [hunk 1]
```diff
@@ -22,31 +22,38 @@ class TSMKeyword {
...
-  new TSMKeyword('MaxSell', 'Max sell price that you sold the item'),
-  new TSMKeyword('BaseItem', 'Reference the base item (without item variations)'),
+  new TSMKeyword('SmartAvgBuy', 'Returns the average purchase price across your last purchases of the item'),
+  new TSMKeyword('AuctioningOpMax', 'Returns the maximum price generated by the operations applied to your groups'),
+  new TSMKeyword('AuctioningOpMin', 'Returns the minimum price generated by the operations applied to your groups'),
+  new TSMKeyword('AuctioningOpNormal', 'Returns the normal price generated by the operations applied to your groups'),
+  new TSMKeyword('AvgBuy', 'Returns the average buy price for all purchases for a particular item'),
+  new TSMKeyword('AvgSell', 'Returns the average sell price for all sales for a particular item'),
+  new TSMKeyword('Crafting', 'Returns the sum of the material prices of each individual item used to craft the recipe'),
+  new TSMKeyword('DBHistorical', 'Returns the 60-day average of the DBMarket value for a particular item on your realms Auction House'),
```
Comment (mike-douglas) [inline src/lib/definitions.js:32]: Could you change "realms" to read "realm\'s" to stay consistent with the other changes you made?

File: src/lib/definitions.js [hunk 2]
```diff
@@ -22,31 +22,38 @@ class TSMKeyword {
...
+  new TSMKeyword('DBMarket', 'Returns the weighted 14-day average of the auctions listed for a particular item on your realms Auction House'),
```
Comment (mike-douglas) [inline src/lib/definitions.js:33]: Same here

File: src/lib/definitions.js [hunk 3]
```diff
@@ -22,31 +22,38 @@ class TSMKeyword {
...
+  new TSMKeyword('DBMinBuyout', 'Returns the lowest priced auction on your realms Auction House'),
```
Comment (mike-douglas) [inline src/lib/definitions.js:34]: And here

**Review by mike-douglas — CHANGES_REQUESTED** [review]
You beat me to doing it! Thanks for submitting, I have a few small requests to keep the wording consistent.

File: src/lib/definitions.js (hunk 1 above)
Comment (jaredycalves) [reply src/lib/definitions.js]: Done

File: src/lib/definitions.js (hunk 2 above)
Comment (jaredycalves) [reply src/lib/definitions.js]: Done

File: src/lib/definitions.js (hunk 3 above)
Comment (jaredycalves) [reply src/lib/definitions.js]: Done

**Review by mike-douglas — APPROVED** [review]
Thanks for taking care of those changes!

**Pull request merged**